frontend/src/js/ui-components/InputDate/CustomHeader.tsx

```diff
@@ -27,12 +25,20 @@ export const SelectMenuContainer = styled("div")`
   width: 100%;
 `;
 
-export const OptionList = styled(List)`
+export const MonthListContainer = styled(List)`
   display: grid;
   grid-template-columns: auto auto;
   gap: 5px;
 `;
 
+export const YearListContainer = styled(List)`
+  display: flex;
+  flex-direction: column-reverse;
+  gap: 5px;
+  height: 200px;
```

Comment:
Is that the picker height? Maybe extract this as a variable?

Reply:
No, it's just the height of the year selection container.
Unfortunately the height of the datepicker is dynamic and can't be extracted that easily.